lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb

```diff
@@ -47,7 +47,9 @@
     You’ll be asked to provide a certificate of no impediment (CNI) to prove you’re allowed to marry. Contact the local notary public to check if you need a CNI.
 
   <% elsif calculator.ceremony_country == 'jordan' %>
-    You need to have been living in <%= calculator.country_name_lowercase_prefix %> for 21 days.
+    <% unless calculator.resident_of_uk? || calculator.resident_of_ceremony_country? %>
+      You need to have been living in <%= calculator.country_name_lowercase_prefix %> for 21 days.
+    <% end %>
 
```

Comment:
I believe you can get rid of this section completely. This outcome is only rendered for residents of the UK or the ceremony country, which means that either `calculator.resident_of_uk?` or `calculator.resident_of_ceremony_country?` will be true; which in turn means that the content in the `unless` block will never be shown.